packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.styles.ts

```diff
@@ -26,6 +26,16 @@ export const getNavbarSwitcherStyles = () => ({
     '&:has(> span:last-child:nth-child(3))': {
       minWidth: '12ch',
     },
+    maxWidth: '15vw',
+    [mqs.xsmall]: {
```

Comment:
unfortunately we can not use `@container` queries as our emotion version is too old. therefore i needed to use 4 different breakpoints to make the environment switcher fit for the different variations